github/repos_pages.go

```diff
@@ -88,3 +92,25 @@ func (s *RepositoriesService) GetLatestPagesBuild(owner string, repo string) (*P
 
 	return build, resp, err
 }
+
+// RequestPageBuild request a build of a GitHub Pages site without needing to push new commit.
```

Comment:
Minor, the second word here should be the verb "request**s**", not the noun "request". RequestPageBuild requests a build .